bin/keycloak-httpd-client-install

```diff
@@ -1090,47 +1205,68 @@ def main():
-            app_name=options.app_name))
-    options.mellon_dst_idp_metadata_file = \
-        os.path.join(options.httpd_saml_dir,
-                     '{app_name}_keycloak_{realm}_idp_metadata.xml'.format(
-                         app_name=options.app_name,
-                         realm=options.keycloak_realm))
-    options.mellon_http_url = \
-        normalize_url('https://{mellon_hostname}:{mellon_https_port}'.format(
-            mellon_hostname=options.mellon_hostname,
-            mellon_https_port=options.mellon_https_port))
-    options.mellon_endpoint_path = join_path(options.mellon_root, options.mellon_endpoint)
-
-    if not options.mellon_entity_id:
-        url = urlsplit(options.mellon_http_url)
-        options.mellon_entity_id = urlunsplit((url.scheme, url.netloc,
-                                               join_path(options.mellon_endpoint_path, 'metadata'),
-                                               '', ''))
+    if options.module == 'mod_auth_mellon':
+        options.mellon_httpd_config_filename = \
+            os.path.join(options.httpd_conf_dir,
+                         '{app_name}_mellon_keycloak_{realm}.conf'.format(
+                             app_name=options.app_name,
+                             realm=options.keycloak_realm))
+        options.mellon_sp_metadata_filename = \
+            os.path.join(options.httpd_dir,
+                         '{app_name}_{mellon_metadata}').format(
+                             app_name=options.app_name,
+                             mellon_metadata=MELLON_METADATA)
+        options.mellon_dst_key_file = \
+            os.path.join(options.httpd_dir,
+                         '{app_name}.key'.format(
+                             app_name=options.app_name))
+        options.mellon_dst_cert_file = \
+            os.path.join(options.httpd_dir, '{app_name}.cert'.format(
+                app_name=options.app_name))
+        options.mellon_dst_idp_metadata_file = \
+            os.path.join(options.httpd_dir,
+                         '{app_name}_keycloak_{realm}_idp_metadata.xml'.format(
+                             app_name=options.app_name,
+                             realm=options.keycloak_realm))
+        options.mellon_http_url = \
+            normalize_url('https://{hostname}:{https_port}'.format(
+                hostname=options.hostname,
+                https_port=options.https_port))
+        options.mellon_endpoint_path = join_path(options.mellon_root, options.mellon_endpoint)
+
+        if not options.mellon_entity_id:
+            url = urlsplit(options.mellon_http_url)
+            options.mellon_entity_id = urlunsplit((url.scheme, url.netloc,
+                                                   join_path(options.mellon_endpoint_path, 'metadata'),
+                                                   '', ''))
+
+    elif options.module == 'mod_auth_openidc':
+        options.mellon_httpd_config_filename = \
```

Comment:
This is the mod_auth_openidc section, why is the argument mellon_httpd_config_filename? Is there any reason both mellon and oidc can't use the same configuration variable, perhaps module_httpd_config_filename